### PR TITLE
CLI flags (fixes #178)

### LINKED ISSUE #178: Add command-line flags for Architect commands
Architect used to rely solely on env vars for CLI configuration; we recently added command line flags for two Architect commands:

- `npx create`
  - `ARC_LOCAL` (env var, boolean)
  - `local`
  - `--local`
  - `-l`

- `npx deploy`
  - `ARC_LOCAL` (env var, boolean)
  - `production`
  - `--production`
  - `-p`
  - `staging`
  - `--staging`
  - `-s`

---

Let's finish it off and add the following flags:
- `npx audit`
  - `apply`
  - `--apply`
  - `-a`

- `npx config`
  - `apply`
  - `--apply`
  - `-a`

- `npx dns`
  - `nuke`
  - `--nuke`
  - `-n`
  - `route53`
  - `--route53`
  - `-r`

- `npx env`
  - `verify`
  - `--verify`
  - `-v`
  - `remove`
  - `--remove`
  - `-r`

- `npx hydrate`
  - `update`
  - `--update`
  - `-u'`

- `npx inventory`
  - `verify`
  - `--verify`
  - `-v`
  - `nuke`
  - `--nuke`
  - `-n`
  - `--nuke=tables`

- `npx sandbox`
  - `production`
  - `--production`
  - `staging`
  - `--staging`
  - `testing`
  - `--testing`

### PR DESCRIPTION
**Pull request by ryanblock**
Thank you for helping out! ✨

Before submitting a pull request, please make sure the you checked the following:

- [x] Forked the repository and created your branch from master
- [x] Make sure the tests pass! `npm it && npm run lint` in the repository root
- [x] If you've fixed a bug or added code **more** tests are appreciated
- [x] If you haven't already please complete the CLA

Learn more about contributing: https://arc.codes/intro/community

### CODE REVIEW

File: src/dns/nuke/index.js [hunk 1]
```diff
@@ -48,7 +49,14 @@ module.exports = function nukeDNS(parms, callback) {
         deleteCNAMERecords,
         deleteHostedZone,
         deleteCertificate,
-      ], callback)
+      ],
+      function done(err, results, callback) {
+        if (err) callback
```
Comment (brianleroux) [inline src/dns/nuke/index.js]: errbacks (where the first param is an error) should not (and mostly will not) pass a callback as a third param. you'll want to both remove that third param and ensure the `if (err) callback(err)` and likewise the else actually invokes the callback on 57 w parens `callback()`

File: src/help/doc/audit.md [hunk 2]
```diff
@@ -5,6 +5,6 @@ Setup a custom IAM Role per Lambda function
 ## Example Usage
 
 -  `npx audit` displays all Lambda functions and associated IAM Role
--  `npx config apply` sync policies found in Lambda function role.json to unique IAM Role
+-  `npx config {apply | --apply | -a}` sync policies found in Lambda function role.json to unique IAM Role
```
Comment (brianleroux) [inline src/help/doc/audit.md]: convention here would be [] brackets for option arguments ala `npx config [apply|--apply|-a]`

File: src/sandbox/env/_env-vars.js [hunk 3]
```diff
@@ -13,23 +13,59 @@ module.exports = function _setupEnv(callback) {
...
+      return c
+    }
+  })
+  command.map(c => {
+    if (Array.isArray(c)) {
+      if (c[0] === '--env' || '-e') {
+        env = c[1]
+      }
+      if (c[0] === '--port' && Number(c[1]) >= 2 && Number(c[1]) <= 65535) {
+        port = Number(c[1])
```
Comment (brianleroux) [inline src/sandbox/env/_env-vars.js]: don't think we should do this right now

File: src/help/doc/audit.md (hunk 2 above)
Comment (ryanblock) [reply src/help/doc/audit.md]: Switched over to []

**Pull request merged**